index.html

```diff
@@ -76,8 +76,8 @@
 
   <a-scene
     renderer="colorManagement: true; physicallyCorrectLights: true; anisotropy: 16; logarithmicDepthBuffer: true;"
-    inspector="url: ./dist/3dstreet-editor.js" timed-inspector="1" loading-screen="enabled: false" notify metadata
-    scene-title reflection device-orientation-permission-ui="enabled: false"
+    loading-screen="enabled: false" notify metadata
```

Comment:
From the inspector component here https://github.com/aframevr/aframe/blob/189993048b4f330872bac0c243a7a3de7415bb0d/src/components/scene/inspector.js#L90 that is auto added, it may call AFRAME.INSPECTOR.open() only initially, not sure.
The logic of the shortcut is there and here https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/index.js#L183-L191

Reply:
@vincentfretin i did notice this issue, but our users will not press ctl-alt-i IRL so I'm ok with the function loss. I think it also better to have our primary js bundle loaded from a traditional script tag at the top of the page instead of in the a-scene tag, as various tooling will expect this.

Reply:
By removing the automatically injected inspector component that register a second keydown listener, it fixes the issue. Only the listener in [3dstreet/src/editor/index.js](https://github.com/3DStreet/3dstreet/blob/2929a6d8b9aa5c1c2b3281fea2eb6659b26929e1/src/editor/index.js#L183-L191) is used and toggle the inspector properly.
#916 may need an additional fix, see my comment there.